app/models/import/person.rb

```diff
@@ -1,5 +1,5 @@
 module Import
-  class Person
+  class Person # rubocop:disable Metrics/ClassLength
```

Comment:
Like you mentioned, this file and UpdateMedicalHistoryFromImport are both long files now. It could possibly be refactored, somehow, to shorten the files, but because the content is very simple to understand I don't feel like refactoring is a very worthwhile use of time at this point. So I think it's fine to just leave it as it is!